src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_utils.py

```diff
@@ -32,8 +33,13 @@ def run_cli_cmd(cmd, retry=0, interval=0, should_retry_func=None):
     try:
         return run_cli_cmd_base(cmd, retry, interval, should_retry_func)
     except CLIInternalError as e:
+        error_code = 'Unknown'
+        error_res = re.search(
```

Comment:
what's this? is there existing method to mask log in cli?

Reply:
It's used to get the error code of azure cli command.  We use subprocess to run other CLI command. The only way to get its error code is to parse the error output.